src/taipy/core/data/sql_db.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2022 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from datetime import datetime, timedelta
+from typing import Dict, List, Optional
+
+from taipy.config.common.scope import Scope
+
+from ..common.alias import DataNodeId, JobId
+from ..exceptions.exceptions import MissingRequiredProperty
+from .sql import SQLDataNode
+
+
+class SQLDbDataNode(SQLDataNode):
+    """Data Node stored in an SQL database.
```

Comment:
"Data Node stored in an SQL database." => "Data Node stored as an SQL database." or something else that you see fit. It would also be great if you can point out the main difference between SQLDbDN and SQLTableDataNode.